apps/masterbots.ai/components/routes/chat/chat.tsx

```
@@ -33,278 +34,263 @@ export function Chat({
-    console.log('Full responses from postICLResponse:', postICLResponse)
-
-    // ! Loading: Now I have the information you need... 'ready'
+        resolve({
+          parsed: chatMetadata,
+          question: userContent,
+          domain: chatbot?.categories[0].category.name as string,
+          chatbot: chatbot?.name as string,
+        });
+        clearTimeout(timeout);
+      }, 700);
+    })) as { parsed: any; question: string; domain: string; chatbot: string };
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)